python/tk_houdini/ui_generation.py

```diff
@@ -961,10 +961,15 @@ def _on_file_change_timeout():
     try:
         tk = sgtk.tank_from_path(cur_file)
     except sgtk.TankError, e:
-        # Unable to get tk api instance from the path. won't be able to get a
-        # new context. if there is an engine running, destroy it.
-        if cur_engine:
-            cur_engine.destroy()
+        
```

Comment:
Why simply not return here ?

In [tk-nuke](https://github.com/shotgunsoftware/tk-nuke/blob/master/python/tk_nuke/__init__.py#L170), if we can't get tank from an unrecognized path, we don't destroy the engine.

Reply:
You're write, it works too!